pep8 test failure on macosx

I can fix this, because I can see the failure. (so, just assign it to me, if you like?)
the pep8 conformance test is failing on my system (macosx mavericks). 

```
======================================================================
FAIL: matplotlib.tests.test_coding_standards.test_pep8_conformance
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/khuff/anaconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/khuff/anaconda/lib/python2.7/site-packages/matplotlib-1.4.x-py2.7-macosx-10.5-x86_64.egg/matplotlib/tests/test_coding_standards.py", line 252, in test_pep8_conformance
    assert_pep8_conformance()
  File "/Users/khuff/anaconda/lib/python2.7/site-packages/matplotlib-1.4.x-py2.7-macosx-10.5-x86_64.egg/matplotlib/tests/test_coding_standards.py", line 236, in assert_pep8_conformance
    assert_equal(result.total_errors, 0, msg)
AssertionError: Found code syntax errors (and warnings):
/Users/khuff/anaconda/lib/python2.7/site-packages/matplotlib-1.4.x-py2.7-macosx-10.5-x86_64.egg/matplotlib/backends/_macosx.py:3:14: E401 multiple imports on one line
/Users/khuff/anaconda/lib/python2.7/site-packages/matplotlib-1.4.x-py2.7-macosx-10.5-x86_64.egg/matplotlib/backends/_macosx.py:4:55: E231 missing whitespace after ','
/Users/khuff/anaconda/lib/python2.7/site-packages/matplotlib-1.4.x-py2.7-macosx-10.5-x86_64.egg/matplotlib/backends/_macosx.py:5:21: E702 multiple statements on one line (semicolon)
/Users/khuff/anaconda/lib/python2.7/site-packages/matplotlib-1.4.x-py2.7-macosx-10.5-x86_64.egg/matplotlib/backends/_macosx.py:6:29: E231 missing whitespace after ','

----------------------------------------------------------------------
Ran 4648 tests in 417.324s

FAILED (KNOWNFAIL=367, failures=1)
```

--- a/lib/matplotlib/tests/test_coding_standards.py
+++ b/lib/matplotlib/tests/test_coding_standards.py
@@ -35,7 +35,8 @@ EXCLUDE_FILES = ['_delaunay.py',
                  '_gtkagg.py',
                  '_backend_gdk.py',
                  'pyparsing*',
-                 '_qhull.py']
+                 '_qhull.py',
+                 '_macosx.py']
 
 PEP8_ADDITIONAL_IGNORE = ['E111',
                           'E112',

